packages/common/src/util/interfaces.ts

```diff
@@ -211,3 +211,14 @@ export interface ICallingCollect {
 }
 
 export interface DeepArray<T> extends Array<T | DeepArray<T>> { }
+
+export interface IRelayConsumerParams {
+  host?: string
+  project: string
+  token: string
+  contexts?: string[]
+  onIncomingCall?: Function
+  onIncomingFax?: Function
```

Comment:
We can remove `onIncomingFax`, since we will not know when we're sending events whether its a fax or not. They'll have to use the `detect` functionality. Eventually we may add a "Fax Service" and this would make sense, but for now we can remove it.

```suggestion
```